e2e/steps_file.js

```diff
@@ -78,6 +81,17 @@ module.exports = function() {
         locate('exui-alert').withText('updated with event: Confirm service'));
     },
 
+    async requestExtension() {
+      await caseViewPage.startEvent('Request extension');
+      await proposeDeadline.enterExtensionProposedDeadline();
+      await extensionAlreadyAgreed.selectAlreadyAgreed();
+
+      await this.retryUntilExists(() => this.click('Ask for extension'), 'ccd-markdown');
+      this.see('You asked for extra time to respond');
+      await this.retryUntilExists(() => this.click('Close and Return to case details'),
+        locate('exui-alert').withText('updated with event: Request extension'));
```

Comment:
I think this assertion should exist in the test file, which it already does! So we could delete this line. What do you think?